src/conn/conn_dhandle.c

```diff
@@ -125,18 +125,13 @@ __wt_conn_dhandle_find(
 static int
 __conn_dhandle_mark_dead(WT_SESSION_IMPL *session)
```

Comment:
Correct, same with the new {{conn_dhandle_clear_open}} they only wrapped the flag manipulation with the exclusive on/off calls.  So those both can go away and the F_SET/F_CLR move back to {{conn_btree_sync_and_close}}